tests/unit/ratelimit_tests.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import datetime
+import unittest
+
+from baseplate import ratelimit
+from pymemcache.client.base import PooledClient
+import pytz
```

Comment:
style nit: I think this should be before any of the `from baseplate import ratelimit`

Reply:
is the idea that `import` goes before `from`?

Reply:
yes

Reply:
pep8 also says local imports should come last, so the baseplate one should be in the final group